api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -9298,5 +9298,33 @@
 			<where>dosing_type = 'FREE_TEXT'</where>
 		</update>
 	</changeSet>
-	
+
+
+	<changeSet id="201604281645" author="vishnuraom">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<sqlCheck expectedResult="0">SELECT count(*) FROM drug_order WHERE dosing_type = 'SIMPLE';</sqlCheck>
+			</not>
+		</preConditions>
+		<comment>Converting values in drug_order.dosing_type column(TRUNK-4845)</comment>
```

Comment:
I think the comment needs to be specific that you're converting for simple dosing type so that it is different from that of the next changeset